packages/quantic/force-app/main/default/lwc/quanticResultAction/quanticResultAction.css

```diff
@@ -1 +1 @@
-@import 'c/quanticResultActionStyles';
\ No newline at end of file
+@import '../quanticResultActionStyles/quanticResultActionStyles.css';
```

Comment:
These 2 modules imports were not working with the c namespace anymore following the update of jest-lwc (probably due to a dependency update that came along with the lwc-jest update.

I tried including the paths in the jsconfig.json but it was not working. (I believe because we are dealing with CSS files). `FacetStyles` is imported with c namespace and working still somehow. Not sure what causes this, read the release notes and nothing jumped to me so its probably a sub-dependency such as @babel

Reply:
Have you checked that the import is still working correctly after this change?

Reply:
yeah its not working in the end, I will revert it to the c namespace and try to figure out something
found a [similar issue](https://developer.salesforce.com/forums/?id=9062I000000ISjwQAG) on salesforce exchange but no resolution